CMakeLists.txt

```diff
@@ -515,10 +515,6 @@ if (EVENT__HAVE_KQUEUE)
     endif()
 endif()
 
-CHECK_SYMBOL_EXISTS(_MINIX "stdio.h" EVENT___MINIX)
-CHECK_SYMBOL_EXISTS(_POSIX_1_SOURCE "stdio.h" EVENT___POSIX_1_SOURCE)
-CHECK_SYMBOL_EXISTS(_POSIX_SOURCE "stdio.h" EVENT___POSIX_SOURCE)
-
```

Comment:
and why did you remove those, because they are unused?

Reply:
My understanding is that the symbols get emitted unmodified by default, but maybe I'm wrong.  I don't use `cmake`.

Reply:
Correct.

Reply:
All I'm saying is that this is cleanup, and we shouldn't mix **fixing real issues** with **cleanups**

Reply:
Okay, so drop these symbols as part of a separate PR or a separate commit?

Reply:
See PR #645.